v1/lib/cell.php

```diff
@@ -45,4 +45,19 @@ public function getCoordX()
     {
         return $this->x;
     }
+
+    public function life()
+    {
+        $this->isAlive = true;
+    }
+
+    public function dead()
```

Comment:
No documentation at all, of course this is not that important. But this is a learning project (!!!), so you need to learn how it works!